### ISSUE TITLE
[Bug]: Does not allow editing Column ID with non-English characters 

### ISSUE THREAD

**Issue by nerkuda** [RafaelGB/obsidian-db-folder#1024]
### Contact Details

nerkuda@gmail.com

### What happened?

In the table settings, I added an existing column, for example, "стоимость". The table shows the values of this column. These values can be edited. But if you open the column settings, you cannot change or save the Column ID. Writes: "Error saving ID"
![BDFolder error 1](https://github.com/RafaelGB/obsidian-db-folder/assets/49241115/d137b165-7174-4a66-b2c8-986cb0bfeee0)

### What platform were you using?

Desktop

### Version of the plugin

3.5.1

### Are you using the latest version of the plugin in your Obsidian vault?

- [X] I have verified that I am on the latest version

### Are you check if there is a similar issue?

- [X] I have verified that there are not similar issues

**Comment by stefandanzl** [comment]
See `src/components/modals/columnSettings/handlers/ColumnIdInputHandler.ts`
```
        // Validate special characters in root key
        if (rootKey.match(/[^a-zA-Z0-9_]/)) {
            return t("column_settings_modal_column_id_error_invalid_key");
        }
```

## Chat GPT:
```
rootKey.match(/[^\w\u0400-\u04FF]/)
```
In this pattern:

`\w` includes word characters (equivalent to [a-zA-Z0-9_]).
`\u0400-\u04FF` includes the Unicode range for Cyrillic characters.
So, [^\w\u0400-\u04FF] will match any character that is not a word character (letters, digits, or underscore) and not a Cyrillic character.

**Comment by stefandanzl** [comment]
@nerkuda Comment some cyrillic ids you'd like me to test

**Comment by stefandanzl** [comment]
`место_покупки` and `место_покупки` do work thusfar

**Comment by stefandanzl** [comment]
Solved in #1023
